bin/acpp

```diff
@@ -925,7 +935,15 @@ class acpp_config:
   def is_pure_linking_stage(self):
     return len(self.source_file_arguments) == 0
 
-def run_or_print(command, print_only):
+def run_or_print(command, print_only, noplugin=False):
+
+  if(noplugin):
+    new_cmd = []
+    for arg in command:
+      if not (arg.startswith("-fplugin=") or arg.startswith("-fpass-plugin")):
+        new_cmd.append(arg)
+    command = new_cmd
+
```

Comment:
If we go with my suggestion, it might be easier to handle this in the `else` branch below (line 949), where we could filter out all unnecessary/uninteresting types of flags from the full command line.